recipe/meta.yaml

```diff
@@ -42,9 +43,9 @@ requirements:
     - mlir {{ version }}
 
 test:
-  imports:
-    - mlir
-    - mlir.ir
+  imports:                  # [build_platform == target_platform]
+    - mlir                  # [build_platform == target_platform]
+    - mlir.ir               # [build_platform == target_platform]
```

Comment:
Leave this out the selectors here, but add
```
test: native_and_emulated
```
to `conda-forge.yml` in the feedstock root.

Reply:
Done in https://github.com/conda-forge/mlir-python-bindings-feedstock/pull/32/commits/0f43913ef8c470741cfb52d776a4be7485986430.